### PR TITLE
setup.py: install kernelci.data package

### PR DESCRIPTION
**Pull request by evdenis**
Otherwise, kci_data fails.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
         "kernelci",
         "kernelci.config",
         "kernelci.lab",
+        "kernelci.data"
```
Comment (gctucker) [inline setup.py:32]:
```suggestion
        "kernelci.data",
```
to avoid making changes to this line again when appending another line.

**Review by gctucker — COMMENTED** [review]
A small tweak, and maybe also add this to the commit message:
```
Fixes: 220eb53ea127 ("setup.py: install kci_data script")
```

File: setup.py (hunk 1 above)
Comment (evdenis) [reply setup.py]: Fixed

**Review by gctucker — APPROVED** [review]
Thanks, tested OK manually after installing the package in `kernelci/build-base` Docker image and running kci_data list_configs from the installed package.

**Pull request merged**